README.md

```diff
@@ -97,6 +102,12 @@ A high level breakdown of the components of the framework:
  * Download release and development [installers and packages](http://www.monogame.net/downloads/).
  * Follow [@MonoGameTeam](https://twitter.com/monogameteam) on Twitter.
 
+## Projects using MonoGame
```

Comment:
I would prefer we don't start maintaining a list like this in the README.

Reply:
Where would you recommend one be maintained then? A separate repo?

Reply:
A list like that is maintained in https://github.com/aloisdeniel/awesome-monogame

Reply:
Oh sweet, thanks! Updating the PR.

Reply:
Can we mark this resolved and get this merged?

Reply:
Tom meant "Projects using MonoGame" shouldn't be in the README at all.